data/moves.ts

```diff
@@ -21725,9 +21731,9 @@ export const Moves: import('../sim/dex-moves').MoveDataTable = {
 				}
 			},
 			onResidualOrder: 4,
-			onResidual(side: any) {
+			onResidual(target: Pokemon) {
 				if (this.getOverflowedTurnCount() <= this.effectState.startingTurn) return;
-				side.removeSlotCondition(this.getAtSlot(this.effectState.sourceSlot), 'wish');
+				target.side.removeSlotCondition(this.getAtSlot(this.effectState.sourceSlot), 'wish');
```

Comment:
It looks as if the changes here made it so that the residual event for a slot now passes the Pokémon in that slot as the target. Does this mean that the call to `removeSlotCondition` can be simplified like this?
```suggestion
				target.side.removeSlotCondition(target, 'wish');
```
If that is indeed the case, would that also prevent the crash in 11147?